### PR TITLE
Removing Apostrophe at Readme start

### PR DESCRIPTION
**Pull request by kunalagra**
Overall looks solid. Minor readme fix i have submitted. ATB for the app.

Also, do considering submitting the app on F-Droid if your are okay with it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-`<h1 align="center">LeetDroid - An android client for LeetCode</h1>
```
Comment (cdhiraj40) [inline README.md:1]: sure, even though the change is small but I would take it. You can revert the changes other than this :)

**Comment by cdhiraj40** [comment]
> Overall looks solid. Minor readme fix i have submitted. ATB for the app.

I am really glad you liked it, thank you!!

> Also, do consider submitting the app on F-Droid if you are okay with it.

Yes sure, I actually never thought about it. But I don't see any harm to that so sure I would upload it :)

File: README.md [hunk 2]
```diff
@@ -51,16 +51,18 @@ If you don't have a leetcode account you can create one from [here](https://leet
 
 ## 🛠️ Installation Steps
 
+- Download from the Playstore or from the Release tab
+
+## 👨‍💻 Contributing
```
[the remark this answers is not in the document]
Comment (kunalagra) [reply README.md]: Sure. Reverted the patch

**Review by cdhiraj40 — APPROVED** [review]
Thanks for your PR! ready to merge :)
